app/src/services/application/Application.js

```diff
@@ -0,0 +1,47 @@
+
+import _ from 'lodash';
+
+import committeesMap from 'common/committees';
+
+export class CommitteeApplication {
+  constructor(id=null, name, email, applicationText, committees, ordered){
+    this._name = name;
+    this._email = email;
+    this._applicationText = applicationText;
+    this._committees = _.clone(committees);
+    this._ordered = ordered;
+  }
+
+  isOrderd(){
+    return this._ordered;
+  }
+
+  get committees(){
+    return this._committees;
+  }
+
+  get applicationText(){
+    return this._applicationText;
+  }
+
+  get name(){
+    return this._name;
+  }
+  
+  get email(){
+    return this._email;
+  }
+
+  get json(){
+    return {
+      "name": this.name,
+      "email": this.email,
+      "application_text": this.applicationText,
+      "prioritized": this.isOrderd(),
+      "committees": this.committees.map((committee, index) => Object.assign({
```

Comment:
I know it's not a change you did, but I don't think Object.assign does anything here.
Remove it, but leave the parenthesis otherwise it won't return an object.